srtcore/group.cpp

```diff
@@ -2656,6 +2656,9 @@ int CUDTGroup::recv(char* buf, int len, SRT_MSGCTRL& w_mc)
                 if (jump > 0)
                 {
                     m_stats.recvDrop.UpdateTimes(jump, avgRcvPacketSize());
+                    LOGC(brlog.Warn,
+                        log << "@" << m_GroupID << " GROUP RCV-DROPPED " << jump << " packet(s): seqno %"
+                        << m_RcvBaseSeqNo << " to " << slowest_kangaroo->second.mctrl.pktseq);
```

Comment:
`" to %"`

(to keep the convention of printing % prefix for sequence numbers

Reply:
Another story: this `if (jump > 0)` looks like a sanity check before updating stats. Normally you should never have a situation that jump <= 0 because if the "slowest kangaroo's" sequence is BASE + 1, you have simply a subsequent packet and not a kangaroo at all.

In result, you should ALWAYS have printed the log present in line 2669, if you had a kangaroo-jump situation.

Reply:
Fixed.